combat-trainer.lic

```diff
@@ -1077,10 +1083,13 @@ class SpellProcess
     @custom_cast = data['cast']
     @symbiosis = data['symbiosis']
     @after = data['after']
+    @skill = data['skill']
+    @prep = data['prep']
     @command = data['command']
     @reset_expire = data['expire'] ? data['abbrev'] : nil
     Flags.reset('ct-spellcast')
     @before = data['before']
+    game_state.action_taken if (@skill == 'Targeted Magic' || @prep == 'target') && game_state.weapon_skill == 'Targeted Magic'
```

Comment:
Do you need the `(@skill == 'Targeted Magic' || @prep == 'target')` part of this conditional? Or, to ask my question differently, what does it add that isn't covered by `game_state.weapon_skill == 'Targeted Magic'`?